.babelrc

```diff
@@ -1,7 +1,4 @@
 {
-  "plugins": [
-    "transform-react-jsx"
```

Comment:
We are you removing this ?

Reply:
This package is not a react package and there is no jsx code used.